village/src/components/SmurfForm.js

```diff
@@ -1,14 +1,10 @@
-import React, { Component } from 'react';
+import React, { Component } from "react";
+import axios from "axios";
 
 class SmurfForm extends Component {
-
   constructor(props) {
     super(props);
-    this.state = {
-      name: '',
-      age: '',
-      height: ''
-    };
+    this.state = { ...this.state };
```

Comment:
How come you didn't set the initial state for `name`, `age`, `height`?

Reply:
It was redundant?